### ISSUE TITLE
Default null input parameter not working in CallActivity

### ISSUE THREAD

**Issue by nhuurnink** [Azure/azure-functions-durable-extension#2703]
When running:
`await context.CallActivityAsync<string>(nameof(MyActivity));`

```
[Function(nameof(MyActivity))]
public Task<string[]> MyActivity([ActivityTrigger] string trigger, FunctionContext functionContext)
```

results in:
` Could not populate the value for 'trigger' parameter. Consider updating the parameter with a default value.`

Which is weird, since CallActivityAsync is supposed to have a default value for `object? input = null`.

**Comment by jviau** [comment]
Most likely related to #2674

**Comment by MatLeger** [comment]
I have the same problem with default null input parameters. I also have the same problem described in #2674.

**Comment by nhuurnink** [comment]
It might be related, but I only have the quotes problem when using DurableTask 1.1.0.

**Comment by jviau** [comment]
Fixed by #2708
